media/js/base/mozilla-fxa-form.js

```diff
@@ -111,6 +111,28 @@ if (typeof window.Mozilla === 'undefined') {
         var serviceField = formElem.querySelector('[name="service"]');
         var userVer = parseFloat(Mozilla.Client._getFirefoxVersion());
 
+
+        var useUITourToSync = userVer > 77 && Mozilla.UITour;
```

Comment:
Would it work to feature-detect the presence of `Mozilla.UITour.showFirefoxAccounts` rather than do a version comparison here?

Reply:
I don't think a feature detect will work here, as `Mozilla.UITour.showFirefoxAccounts` is just a JS lib as opposed to a browser feature.